handlers/main.yml

```diff
@@ -195,6 +195,12 @@
     name: NetworkManager
     state: restarted
 
+- name: Reload NetworkManager
```

Comment:
Can you add a comment why we're doing reload after restart?

Reply:
Done